# Either orientation or rpy must be spefiyed
# Make overwrite true to make previous motion skip
bool overwrite

geometry_msgs/Point pos  #  [mm]

geometry_msgs/Quaternion orientation  # quaternion
geometry_msgs/Vector3 rpy  # rpy[degree]
float64 fig  #fig

uint8 comp  # interpolation methods
#interpolation enum
uint8 PTP=1
uint8 CP=2
---
uint8 SUCCESS=0
uint8 OUTRANGE=1
uint8 status
